src/ussd_clinic_rapidpro.js

```diff
@@ -1205,7 +1205,7 @@ go.app = function() {
             };
             var flow_uuid;
 
-            if (self.im.user.answers.state_message_type === "state_edd_month"
+            if (self.im.user.answers.state_message_type === "state_edd_year"
```

Comment:
Why this change?

Reply:
state_message_type choice option are _state_edd_year_ for prebirth and _state_birth_year_ for postbirth. https://github.com/praekeltfoundation/ndoh-jsbox/blob/develop/src/ussd_clinic_rapidpro.js#L354

Reply:
But this is in the clinic line? The ticket is for baby dob change on the popi line?

Reply:
Cool i will revert these changes